CMakeLists.txt

```diff
@@ -30,6 +30,8 @@ option(ACTS_BUILD_EXAMPLES "Build examples" OFF)
 option(ACTS_BUILD_UNITTESTS "Build unit tests" OFF)
 option(ACTS_BUILD_INTEGRATIONTESTS "Build integration tests" OFF)
 option(ACTS_BUILD_DOC "Build documentation" OFF)
+option(ACTS_BUILD_CUDA_FEATURES "Build cuda-related features" OFF)
```

Comment:
We usually call these 'plugins' if they have an external dependency, i.e. the ROOT plugin.